phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.css

```diff
@@ -644,6 +644,11 @@
 /************************ icons ************************/
 /*******************************************************/
 
+/************************ util *************************/
+.refreshIcon {
+    -fx-graphic: url('/org/phoenicis/javafx/themes/breezeDark/icons/mainwindow/refresh.png');
```

Comment:
This should be removed (it doesn't work currently and does not belong to the topic of this PR at all).